src/main/channels/audio/DeleteOneAudioChannel.ts

```
@@ -1,20 +1,18 @@
+import { getUserAudioPath } from '@main/utils/path';
 import { DELETE_ONE_AUDIO_CHANNEL } from '@shared/channels';
 import { DeleteOneAudioParams } from '@shared/interfaces/Audio';
 import { IpcRequest } from '@shared/interfaces/IpcRequest';
 import { IpcMainEvent } from 'electron';
 import log from 'electron-log/main';
-import * as fs from 'fs';
-import * as path from 'path';
+import fs from 'fs';
+import path from 'path';
 
 function deleteAudio(filePath: string) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)